worlds/seaofthieves/Rules.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+from worlds.seaofthieves.Items.Items import Items
```

Comment:
```suggestion
from .Items.Items import Items
```
Use relative imports for imports within your world